lib/SILOptimizer/IPO/CrossModuleOptimization.cpp

```diff
@@ -730,7 +778,7 @@ void CrossModuleOptimization::serializeFunction(SILFunction *function,
   }
   function->setSerializedKind(getRightSerializedKind(M));
 
-  InstructionVisitor visitor(*function, *this);
+  InstructionVisitor visitor(*function, *this, false);
```

Comment:
Please add an "argument label" here: `InstructionVisitor visitor(*function, *this, /*onlyCheckIfSerializable=*/ false)`